<launch>

    <arg name="publish_robot_source_odom_tf" default="true"/>
    <node name="joy_node" pkg = "joy" type="joy_node" />
    <node name="base_Only" pkg="basecontrol" type="baseOnly" output="screen" required="true">
        <param name="base_foot_print" value="base_link"/>
        <param name="odom_frame" value="odom"/>
        <param name="publish_tf" value="$(arg publish_robot_source_odom_tf)"/>
        <param name="serial_addr" value="/dev/ttyUSB0"/>
    </node>

    
</launch>
